TLM/TLM/Lifecycle/TMPELifecycle.cs

```diff
@@ -62,7 +62,8 @@ public class TMPELifecycle : MonoBehaviour {
         /// <summary>
         /// determines whether Game mode as oppose to edit mode (eg asset editor).
         /// </summary>
-        internal static bool PlayMode => SceneManager.GetActiveScene().name.Equals("InGame");
+        internal static bool PlayMode => SceneManager.GetActiveScene().name.Equals("Ingame") ||
```

Comment:
Twice accessing global.function().field, can this be expressed with a temporary variable for name?